### PR TITLE
Added my solution to the FizzBuzz problem, and added a link to my readme

### PR DESCRIPTION
**Pull request by gottamike**

### CODE REVIEW

File: fizzbuzz/java/mikegotta/MikesREADME.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Mike's FizzBuzz
+
+Assignment 3 
+-----------------------------------
+How to run Mikes FizzBuzz solution:
+- navigate to the directory fizzbuzz/java/mikegotta
+- change your permission with:  chmod +x runner.sh
```
Comment (turnerburchard) [inline fizzbuzz/java/mikegotta/MikesREADME.md:7]: I think this should say runFizzBuzz.sh

**Review by turnerburchard — COMMENTED** [review]
I ran "./runFizzBuzz 17" with error output:

`./runFizzBuzz.sh: line 2: $'\r': command not found
"xception in thread "main" java.lang.NumberFormatException: For input string: "17
        at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:68)
        at java.base/java.lang.Integer.parseInt(Integer.java:652)
        at java.base/java.lang.Integer.parseInt(Integer.java:770)
        at FizzBuzz.main(FizzBuzz.java:38)`

File: fizzbuzz/java/mikegotta/MikesREADME.md (hunk 1 above)
Comment (gottamike) [reply fizzbuzz/java/mikegotta/MikesREADME.md]: Hey Turner, I fixed that bug in the readme. As for the error you are getting... apparently it has something to do with java not being able to turn the string "17" into the integer "17". It works on my end, maybe I have a different version of java then you?

**Review by abbykq — CHANGES_REQUESTED** [review]
+10
I ran your code and it runs great! Just one thing, like Turner said for line 7  in your read me you say to change your permission with:  chmod +x runner.sh but your run file is named runFizzBuzz.sh so just update your readme file to say chmod +x runFizzBuzz.sh instead :)
mikegottaFizzBuzz

**Comment by gottamike** [comment]
> +10 I ran your code and it runs great! Just one thing, like Turner said for line 7 in your read me you say to change your permission with: chmod +x runner.sh but your run file is named runFizzBuzz.sh so just update your readme file to say chmod +x runFizzBuzz.sh instead :) mikegottaFizzBuzz

I went ahead and fixed that README bug, it should be correct now

**Review by abbykq — APPROVED** [review]

**Comment by Phred7** [comment]
Looks good to me +1

**Pull request merged**